checker/src/main/java/org/checkerframework/checker/nullness/NullnessNoInitAnnotatedTypeFactory.java

```diff
@@ -113,6 +113,9 @@ public class NullnessNoInitAnnotatedTypeFactory
     private final ExecutableElement mapGet =
             TreeUtils.getMethod("java.util.Map", "get", 1, processingEnv);
 
+    // High-level optimization: cache entire addComputedTypeAnnotations processing flow
+    private final java.util.Set<String> fullyProcessedTrees = new java.util.HashSet<>();
```

Comment:
Why not import Set and HashSet?
At least Set is already imported. We already use Linked HashSet, which I think has better deterministic behavior compared to HashSet.